com.github.onegal.Viper.yaml

```diff
@@ -0,0 +1,57 @@
+app-id: com.github.onegal.Viper
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
```

Comment:
```suggestion
base-version: '21.08'
```

Reply:
Updated in 6f3cce50d3e2b75e910d92a8c2b359f030ab55fb